programs/stake_api/src/stake_instruction.rs

```diff
@@ -31,6 +31,22 @@ pub enum StakeInstruction {
     ///    2 - RewardsPool Stake Account from which to redeem credits
     ///    3 - Rewards syscall Account that carries points values
     RedeemVoteCredits,
+
+    /// Deactivates the stake in the account
+    ///
+    /// Expects 1 Accounts:
+    ///    0 - Delegate StakeAccount
```

Comment:
need 2: the delegate and syscall::current

Reply:
👍